keepsorted/block.go

```diff
@@ -68,9 +67,10 @@ const (
 // include is a function that lets the caller determine if a particular block
 // should be included in the result. Mostly useful for filtering keep-sorted
 // blocks to just the ones that were modified by the currently CL.
-func (f *Fixer) newBlocks(lines []string, offset int, include func(start, end int) bool) ([]block, []incompleteBlock) {
+func (f *Fixer) newBlocks(lines []string, offset int, include func(start, end int) bool) ([]block, []incompleteBlock, map[int]optionWarnings) {
```

Comment:
Since we're already passing the file name into a few new places in this PR, should we perhaps just pass it into this function, too? Then I believe we can generate `Finding`s directly instead of making this `map[int]optionWarnings` that gets post-processed into `Finding`s.